lib/firefox/geckoProfilerDefaults.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 module.exports = {
-  features: 'js,stackwalk,leaf,responsiveness',
+  features: 'js,stackwalk,leaf',
```

Comment:
There is a trailing comma here that wasn't here before and I don't know how well we handle that in the API.

Reply:
Nevermind, I misread this :)